src/routes/api.ts

```diff
@@ -0,0 +1,37 @@
+import { FastifyInstance, FastifyReply, FastifyRequest } from 'fastify';
+import ConfigFactory from '../config/config-factory';
+import { Config, ConfigSystem } from '../config';
+
+export default async function api(fastify: FastifyInstance) {
+  fastify.get('/api', (req: FastifyRequest, reply: FastifyReply) => {
+    reply.status(200).send({
+      name: 'UMT API',
+      version: '1.0',
+      status: 'healthy'
+    });
+  });
+
+  fastify.get('/api/config', async (req: FastifyRequest, reply: FastifyReply) => {
+    try {
+      const { config } = ConfigFactory.getConfigFactory().loadConfig();
+      reply.status(200);
+      return config;
+    } catch (error) {
+      reply.status(404).send({ message: 'No configuration found' });
```

Comment:
i dont think 404 is the right response.  that would imply the `/api/config` endpoing wasn't found, but it was found... it just just returning empty.  i think 200 with empty response or message is better